TodoApp/TodoApp.Api.Tests/ItemsControllerTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+using NUnit.Framework;
+using TodoApp.Api.Controllers;
+using TodoApp.Api.Models;
+using TodoApp.Api.Tests.Utils;
+
+namespace TodoApp.Api.Tests
+{
+    [TestFixture]
+    public class ItemsControllerTests
+    {
+        private ItemsController _itemsController;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _itemsController = new ItemsController
+            {
+                Request = new HttpRequestMessage(),
+                Configuration = new HttpConfiguration()
+            };
+        }
+
+        [Test]
+        public async Task GetAllItemsAsync_ReturnsAllItems()
+        {
+            var message = await ExecuteAsyncAction(async () => await _itemsController.GetAllItemsAsync());
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(message.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+                Assert.That(message.Content, Is.Not.Null);
+            });
+        }
+
+        [Test]
+        public async Task GetItemByIdAsync_ReturnsOkWithRequiredItem()
+        {
+            const string id = "1";
+
+            var message = await ExecuteAsyncAction(async () => await _itemsController.GetItemByIdAsync(id));
+
+            Assert.Multiple(() =>
+            {
+                Assert.That(message.StatusCode, Is.EqualTo(HttpStatusCode.OK));
+                Assert.That(message.Content, Is.Not.Null);
+                Assert.That(message.TryGetContentValue<Item>(out var item), Is.Not.Null);
```

Comment:
`message.TryGetContentValue<Item>` should not be part of assert. Move it after you get the message as it should be part of _ACT_ phase of the test.
Validate in all other tests.